Button guidelines order (#2729)
coveo/plasma: packages/website/docs/Button.md

> a# Best Practices

Reviewer: ```suggestion
# Best Practices
```

Similarly, in the footer of a [modal wizard](https://plasma.coveo.com/layout/ModalWizard), navigation buttons must be ordered as follows: "Previous" (secondary), then "Next" (primary). At the last step, the “Next” button must be replaced with a primary button with a label indicating the resulting action. For example, an appropriate label would be “Add filter” rather than “Save”.

> For buttons located in a [page header](https://plasma.coveo.com/layout/PageHeader), always put **the primary button to the left** instead and the secondary to the right of the primary button. In a page header, the primary action usually relates to the creation of new elements, while secondary actions are usually used to trigger troubleshooting or management actions, such as activity review. If multiple secondary actions are required, regroup them using a [actionable item](https://plasma.coveo.com/form/ActionableItem) on the rightmost side of the header. 

Reviewer: Is it possible to have several primary buttons in the same page?

Reviewer 2: ```suggestion
In a [page header](https://plasma.coveo.com/layout/PageHeader), **the primary button should appear on the left** and the secondary button should be to the right of the primary button. The primary action usually relates to the creation of new elements, while secondary actions are usually used to trigger troubleshooting or management actions, such as activity review. If multiple secondary actions are required, group them using an [actionable item](https://plasma.coveo.com/form/ActionableItem) on the rightmost side of the header. 
```
